### ISSUE TITLE
Fix load balancer returning 502 Bad Gateway error

### ISSUE THREAD

**Issue by Siegrift** [api3dao/signed-api#211]
One of the Airnode feed issues is 502 Gateway rejection by AWS ELB. See the query [here](https://api3.grafana.net/explore?schemaVersion=1&panes=%7B%22cot%22:%7B%22datasource%22:%22grafanacloud-logs%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22expr%22:%22%7Bapp%3D%5C%22airnode-feed%5C%22%7D%20%7C%3D%20%60Request%20failed%20with%20status%20code%20502%60%22,%22queryType%22:%22range%22,%22datasource%22:%7B%22type%22:%22loki%22,%22uid%22:%22grafanacloud-logs%22%7D,%22editorMode%22:%22builder%22,%22maxLines%22:100%7D%5D,%22range%22:%7B%22from%22:%22now-30m%22,%22to%22:%22now%22%7D%7D%7D&orgId=1).

The Signed APIs seem to be underutilised based of the CPU and memory metrics. This should be investigated and fixed. Note, that these rejections happen sporadicly and doesn't seem to impact the system that much.

**Comment by Siegrift** [comment]
I enabled access and connection load balancer logs on one of the Signed APIs and hope to get more clarity. The Signed API service seems to be under-used (max ~15% CPU and MEM according to the dashboard).

**Comment by Siegrift** [comment]
I gathered some of the [ELB logs](https://s3.console.aws.amazon.com/s3/buckets/ncfx-alb-access-logs?bucketType=general&prefix=AWSLogs%2F715606441724%2Felasticloadbalancing%2Fus-east-1%2F2024%2F02%2F02%2F&region=us-east-1#) and compared it with [Grafana logs](https://api3.grafana.net/explore?schemaVersion=1&panes=%7B%22azp%22%3A%7B%22datasource%22%3A%22grafanacloud-logs%22%2C%22queries%22%3A%5B%7B%22refId%22%3A%22A%22%2C%22expr%22%3A%22%7Bapp%3D%5C%22airnode-feed%5C%22%2C+airnode%3D%5C%220xbC6471E88d8aFe936A45bEB8bd20a210EBEF6822%5C%22%7D+%21%3D+%60debug%60%22%2C%22queryType%22%3A%22range%22%2C%22datasource%22%3A%7B%22type%22%3A%22loki%22%2C%22uid%22%3A%22grafanacloud-logs%22%7D%2C%22editorMode%22%3A%22builder%22%7D%5D%2C%22range%22%3A%7B%22from%22%3A%221706888488000%22%2C%22to%22%3A%221706888525000%22%7D%7D%7D&orgId=1). Note, that I turned off the ELB logs after a while because they are extra costs.

The ELB logs are nice and provide a lot of information. I saw that few requests were rejected by ELB. I was able to match other (successful) request with Grafana logs. I couldn't find **why** they are rejected though. I observed, that before each 502 rejection Airnode feed pushed large batch of Signed data (size 100), so my hypothesis is that signature verification caused a very short term CPU spike which caused ELB to reject the other request.

Note, that the failure happens infrequently, but I want us to understand the reason behind every error we see. I think https://github.com/api3dao/signed-api/issues/213 and https://github.com/api3dao/signed-api/issues/212 could both help with mitigating this issue.

As an unrelated note, I noticed that some of the Grafana logs look quite strange:
```
{
  "log": "...."
  "partial_id": "....",
  "partial_last": "false",
  "partial_message": "true",
  "partial_ordinal": "2"
}
```

This is caused by the [log forwarder](https://github.com/fluent/fluent-bit/issues/4309) when it needs to forward long lines (it splits them into these partial logs). We should not see this with INFO level.

**Comment by Siegrift** [comment]
https://github.com/api3dao/signed-api/pull/215 might fix this issue. We need to redeploy the Signed APIs and see if there are  less errors for a few days.

**Comment by Siegrift** [comment]
I've checked Grafana logs after redeploing with 0.5.1. Before the mentioned fix, there were ~300 errors during a 12 hour period, but after the fix its ~90 which is significant reduction, but it's not a complete fix.

I also noticed that 0.6.0 has more of these errors, but @bdrhn9 pointed out this may be because of the increased bandwith from logging the data.

I think this is enough to warrant closing the issue. To get rid of 502 errors completely, we may get rid of ELB.
